ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx

```diff
@@ -137,6 +139,31 @@ enum Mode {
   BadInput = 'bad-input',
 }
 
+const handleMetacardUpdate = (
```

Comment:
I like to stick to one parameter, but send things in as an object.  I find it simpler to maintain.